schema/sale_artwork.js

```diff
@@ -178,8 +178,12 @@ const SaleArtworkType = new GraphQLObjectType({
         type: GraphQLBoolean,
         resolve: ({ reserve_status }) => reserve_status !== 'no_reserve',
       },
+      lot_label: {
+        type: GraphQLString,
+      },
       lot_number: {
         type: GraphQLString,
+        deprecationReason: 'Favor `lot_label`',
```

Comment:
Got rid of the field switch since I plan to update clients anyways. This is less dangerous in case there's some odd type thing happening on their side that I'm unaware of.